src/ext/BalExtension/mba/core/EventArgs.cs

```diff
@@ -416,19 +416,20 @@ public string Content
     /// Additional arguments used when the detection for an update has completed.
     /// </summary>
     [Serializable]
-    public class DetectUpdateCompleteEventArgs : StatusEventArgs
+    public class DetectUpdateCompleteEventArgs : ResultStatusEventArgs
     {
         private string updateLocation;
-
+        
```

Comment:
Yep, I need to find out what keeps injecting whitespace, as I don't think it's me.